docs/tutorials/transforms.ipynb

```diff
@@ -703,39 +696,67 @@
       "text/plain": [
        "<PIL.Image.Image image mode=RGB size=256x256 at 0x7F009BE81F10>"
       ]
      },
+     "execution_count": 20,
      "metadata": {},
-     "execution_count": 20
+     "output_type": "execute_result"
     }
+   ],
+   "source": [
+    "dataset = EuroSAT(transforms=MinMaxNormalize(mins, maxs))\n",
+    "\n",
+    "# @title EuroSat Multispectral (MS) Browser  { run: \"auto\", vertical-output: true }\n",
+    "idx = 16199  # @param {type:\"slider\", min:0, max:16199, step:1}\n",
+    "sample = dataset[idx]\n",
+    "rgb = sample[\"image\"][1:4]\n",
+    "rgb = rgb[[2, 1, 0], ...]\n",
+    "image = T.ToPILImage()(rgb)\n",
+    "print(f\"Class Label: {dataset.classes[sample['label']]}\")\n",
+    "image.resize((256, 256), resample=Image.BILINEAR)"
    ]
   }
- ]
+ ],
+ "metadata": {
+  "accelerator": "GPU",
+  "colab": {
+   "name": "indices.ipynb",
```

Comment:
So THIS is why all of the tutorials have the wrong filename when I open them in Colab! I'll fix the other tutorials in a separate PR.
```suggestion
   "name": "transforms.ipynb",
```